frontend/wallet/src/DaemonProcess.re

```diff
@@ -0,0 +1,45 @@
+open Bindings;
+
+let process: ref(option(ChildProcess.Process.t)) = ref(None);
```

Comment:
I don't like having this ref. Is it possible to code this without having this ref?